repos/javascriptservices.proj

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
   <PropertyGroup>
-    <BuildCommand>$(DotnetToolCommand) pack -c $(Configuration) --no-build Microsoft.DotNet.Web.Spa.ProjectTemplates.csproj /p:NuspecFile=Microsoft.DotNet.Web.Spa.ProjectTemplates.nuspec</BuildCommand>
-    <PackagesOutput>$(ProjectDirectory)/bin/$(Configuration)/</PackagesOutput>
+    <BuildCommand>$(DotnetToolCommand) pack -c $(Configuration) --output $(SourceBuiltPackagesPath) --no-build Microsoft.DotNet.Web.Spa.ProjectTemplates.csproj /p:NuspecFile=Microsoft.DotNet.Web.Spa.ProjectTemplates.nuspec /p:MSBuildProjectExtensionsPath=$(IntermediatePath)$(RepositoryName)/ /p:IntermediateOutputPath=$(IntermediatePath)$(RepositoryName)/ /p:OutputPath=$(OutputPath)$(RepositoryName)/ /p:RestoreOutputPath=$(IntermediatePath)$(RepositoryName)</BuildCommand>
```

Comment:
I think you can just set `BaseIntermediateOutputPath` instead of having to set `MSBuildProjectExtensionsPath`, `IntermediateOutputPath`, `RestoreOutputPath`.

Reply:
Yes.  I changed it to use BaseIntermediateOutputPath.